src/components/settings/Users/EditUser.vue

```diff
@@ -0,0 +1,219 @@
+                <label class="form-label d-block">
+                    {{ $t("New Password") }}
+                    <input
+                        v-model="newPassword"
+                        type="password"
+                        class="form-control mt-2"
+                        required
+                        :disabled="savingPassword"
+                    />
+                </label>
+            </div>
+
+            <div class="mb-3">
+                <label class="form-label d-block">
+                    {{ $t("Repeat New Password") }}
+                    <input
+                        v-model="repeatNewPassword"
+                        type="password"
+                        class="form-control mt-2"
+                        required
+                        :disabled="savingPassword"
+                    />
+                </label>
+            </div>
+
+            <button class="btn btn-primary" type="submit" :disabled="savingPassword">
+                <span v-show="savingPassword" class="spinner-border spinner-border-sm" role="status" aria-hidden="true" />
+                {{ $t("Update Password") }}
+            </button>
+        </form>
+
+        <h5 class="mt-5 mb-4 settings-subheading">{{ $t("Permissions") }}</h5>
+        <div class="form-check form-switch">
+            <label class="form-check-label">
+                <input
+                    :checked="active"
+                    class="form-check-input"
+                    style="scale: 1.4; cursor:pointer"
+                    type="checkbox"
+                    :disabled="saving"
+                    @click="debounce(() => { active = !active; save({ active }); })"
+                >
+                <div class="ps-2">{{ $t("Active") }}</div>
+            </label>
+        </div>
+    </template>
+</template>
+
+<script>
+import { useToast } from "vue-toastification";
+const toast = useToast();
+
+/**
+ * Get debounced function
+ * @returns {function} debounced function
+ */
+function Debounce() {
+    let timeout = null;
+
+    /**
+     * exec callback function after delay if no new call to function happens
+     * @param {function} callback function to execute after delay
+     * @param {number} [delay=100] delay before execute the callback if no new call to function happens
+     * @returns {void}
+     */
+    return function (callback, delay = 100) {
+        clearTimeout(timeout);
+        timeout = setTimeout(() => callback(), delay);
+    };
+}
+
+export default {
+    props: {
+        id: {
+            type: String,
+            required: true
+        }
+    },
+
+    data: () => ({
+        loading: false,
+        username: "",
+        saving: false,
+        currentPassword: "",
+        newPassword: "",
+        repeatNewPassword: "",
+        savingPassword: false,
+        active: false
+    }),
+
+    mounted() {
+        this.getUser();
+    },
+
+    methods: {
+        // Used to prevent dbl "click" evt fired by checkbox label
```

Comment:
what is dbl?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event

Reply:
Reason behind that debounce needs it when the user is clicking (single click) on the switch (which is in fact a modified checkbox) label. Sometimes (dunno why 🤷🏼‍♂️), two "single click" events are fired... So I debounce the event handler to workaround that issue.

Reply:
```suggestion
        // Used to prevent the "dblclick" event being fired by the checkbox label
```

Reply:
Mmmm it will be even more confusing I guess because no "dblclick" event is fired... 
Perhaps it may be clearer with:
```
// Used to ignore one of the two "click" event fired when clicking on checkbox label
```
wdyt?

Reply:
Way better 👍🏻 

two grammar corrections:
```suggestion
        // Used to ignore one of the two "click" events fired when clicking on the checkbox label
```

Reply:
Done, see last commit